src/main/resources/messages/LanguageServerBundle.properties

```diff
@@ -33,10 +33,10 @@ language.server.mappings.fileNamePattern.no=No file name patterns mappings confi
 language.server.mappings.languageId.column=Language Id
 
 language.server.tab.configuration=Configuration
+language.server.tab.configuration.server=Server configuration
+language.server.tab.configuration.client=Client configuration
 language.server.configuration=Configuration:
-language.server.configuration.emptyText=Defines the configuration of the language server

```

Comment:
These two messages were unused after the move to `JsonTextField` which doesn't support placeholder text.